examples/call_flow.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import sys
+import messagebird
+import pprint
+
+#ACCESS_KEY = ''
+#CALL_FLOW_ID = ''
+
+try:
+    ACCESS_KEY
+except NameError:
+    print('You need to set an ACCESS_KEY constant in this file')
+    sys.exit(1)
+
+try:
+    CALL_FLOW_ID
+except NameError:
+    print('You need to set a CALL_FLOW_ID constant in this file')
+    sys.exit(1)
+
+try:
+    # Create a MessageBird client with the specified ACCESS_KEY.
+    client = messagebird.Client(ACCESS_KEY)
+
+    # Fetch the Message object for the specified MESSAGE_ID.
+    call = client.call_flow(CALL_FLOW_ID)
+
+    pp = pprint.PrettyPrinter(indent=4)
+
+    # Print the object information.
+    print('\nThe following information was returned as a Message object:\n')
+    print('  id           : %s' % call.id)
```

Comment:
Please consider to use `'{}'.format(call.id)` instead of `%`. https://pyformat.info/